pkg/fronius/symo.go

```diff
@@ -86,18 +88,34 @@ type (
 		Values map[string]float64
 	}
 
+	SymoMeter struct {
+		Body struct {
+			Data map[string]MeterData
+		}
+	}
+
+	MeterData struct {
+		// "EnergyReal_WAC_Sum_Consumed": 560839.0,
+		// "EnergyReal_WAC_Sum_Produced": 94087.0,
+		EnergyRealSumConsumed float64 `json:"EnergyReal_WAC_Sum_Consumed"`
+		EnergyRealSumProduced float64 `json:"EnergyReal_WAC_Sum_Produced"`
+	}
+
 	// SymoClient is a wrapper for making API requests against a Fronius Symo device.
 	SymoClient struct {
 		request *http.Request
 		Options ClientOptions
 	}
 	// ClientOptions holds some parameters for the SymoClient.
 	ClientOptions struct {
-		URL              string
-		Headers          http.Header
-		Timeout          time.Duration
-		PowerFlowEnabled bool
-		ArchiveEnabled   bool
+		URL               string
+		Headers           http.Header
+		Timeout           time.Duration
+		PowerFlowEnabled  bool
+		ArchiveEnabled    bool
+		SmartMeterEnabled bool
+		OffsetConsumed    float64
+		OffsetProduced    float64
```

Comment:
From this naming alone it's not clear to me what it means or when it's relevant. Please either try to find a better name, or add field comments describing its effects.